### PR TITLE
Cs 807 gov uk footer links

### PR DESCRIPTION
**Pull request by Chazzysb**
Cs 807 gov uk footer links

### CODE REVIEW

File: src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Footer.cshtml [hunk 1]
```diff
@@ -5,8 +5,14 @@
             <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
                 <h2 class="govuk-visually-hidden">Support links</h2>
                 <ul class="govuk-footer__inline-list">
+                    <li class="govuk-footer__inline-list-item">                        
+                        <a class="govuk-footer__link" href="/Privacy">Privacy</a>
```
Comment (benagain) [inline src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Footer.cshtml:9]: These will work when rendered as part of the Portal website, as the relative URL will be right for the portal.  When rendered from the Shared UI in the Commitments Web the relative URL will end up being "confirm.../Privacy", which won't exist.

There is a helper tag `asp-external-section` that takes care of this:
```suggestion
                        <a class="govuk-footer__link" asp-external-section="@NavigationSection.Home" asp-external-page="Privacy">Privacy</a>
```

File: src/SFA.DAS.ApprenticePortal.Web/Pages/Privacy.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+namespace SFA.DAS.ApprenticePortal.Web.Pages
+{
+    public class PrivacyModel : PageModel
+    {
+        /// <summary>
+        /// Page yet to be implemented
+        /// </summary>
+        public void OnGet()
+        {
+        }
```
Comment (benagain) [inline src/SFA.DAS.ApprenticePortal.Web/Pages/Privacy.cshtml.cs:12]: Empty methods cause SonarCloud to complain.  As there is no logic in these pages we can actually just delete the `.cshtml` files completely.

File: src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Footer.cshtml (hunk 1 above)
Comment (Chazzysb) [reply src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Footer.cshtml]: Updated to use NavigationSection

File: src/SFA.DAS.ApprenticePortal.Web/Pages/Privacy.cshtml.cs (hunk 2 above)
Comment (Chazzysb) [reply src/SFA.DAS.ApprenticePortal.Web/Pages/Privacy.cshtml.cs]: Removed code behind files

**Review by benagain — APPROVED** [review]

**Pull request merged**